tests/tests_configure_ha_cluster.yml

```diff
@@ -46,7 +46,10 @@
     - name: Set up test environment for the ha_cluster role
       include_role:
         name: fedora.linux_system_roles.ha_cluster
-        tasks_from: test_setup.yml
+        tasks_from: "{{ item }}"
+      loop:
+        - test_setup.yml
+        - test_setup_sbd.yml
```

Comment:
`test_cleanup_sbd.yml` doesn't seem to be called anywhere. This may not be an issue for CI systems which spin up new VMs / containers for every test, though.

Reply:
Right, mssql does not have proper clean_up playbooks to run in CI, however, there is tests/clean_up_mssql_pacemaker.yml that I use to clean up systems when testing locally. I mostly test against VMs because I need multi-node testing.
Also, most of the time I run the tests/tests_configure_ha_cluster.yml to configure SQL HA and continue developing on it. I then would run tests/clean_up_mssql_pacemaker.yml to remove configuration and start again. Because test and clean up plays are not executed within a single playbook, it's not possible to share facts between them, so I took a different approach. 
Please check the latest commit https://github.com/linux-system-roles/mssql/pull/83/commits/5dc1fc63076410a60b608c66ec3ecc0869b3e3c4 where I implement clean up.

Reply:
> I think that's fine for now.

Agreed.